docs/odissei-tutorial.md

```diff
@@ -0,0 +1,137 @@
+
+# How to use the `sirup` package to manage your IP address in python
+
+When scraping data from the internet, it can be handy to use an IP address that is different from your real IP address. The `sirup` package in python allows you to do this in a flexible way through a VPN service.
```

Comment:
"it can be handy" -> give some concrete reasons why you need to do this e.g. privacy